ci/kind/kind-setup.sh

```diff
@@ -376,7 +376,7 @@ function create {
   fi
 
   set +e
-  kind get clusters | grep $CLUSTER_NAME > /dev/null 2>&1
+  kind get clusters | grep -w "^$CLUSTER_NAME$" > /dev/null 2>&1
```

Comment:
What's the purpose of `-w` here if you add `^` and `$` to the regex?
Looking at the grep manual, it feel like `grep -x "$CLUSTER_NAME"` / `grep --line-regexp"$CLUSTER_NAME"` would be more appropriate?

Reply:
I added it for more clarity, but i think can use -x also.